core/server/utils/parse-package-json.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
```

Comment:
afaik, this is the default and therefore not required (we removed these from the core code many moons ago)